### DISCUSSION TITLE
Meetupy pazdziernik

### DISCUSSION THREAD

**Discussion by mgrabarz** [AzureCommunityPL/group#462] [Wydarzenia]
Cześć,
MAUGP w tym miesiącu planuje sporo wydarzeń, wpadajcie śmiało na spotkania!

📅 **12 października 18:00-21:00**, [#3 spotkanie Microsoft Azure User Group Poland w Rzeszowie](https://www.meetup.com/pl-PL/microsoft-azure-users-group-poland/events/288500280/)
Wystąpią @jlamch @mimachniak

📅 **12 października 18:00-21:00**, [#52 spotkanie Microsoft Azure User Group Poland w Białymstoku](https://www.meetup.com/pl-PL/microsoft-azure-users-group-poland/events/288931668/)
Wystąpią Patrycjusz Jaskurzyński oraz Paweł Szczecki

📅 **19 października 18:00-21:00**, [Ignite After Party w Warszawie](https://www.meetup.com/pl-PL/microsoft-azure-users-group-poland/events/288897427/)
Wystąpią: @wi5nia @eklime @kaluzaaa Kamil Bączyk oraz @michalfurmankiewicz

**Comment by techitlab** [comment]
Dzieki za spotkanie i prezentacje.  Bylo mega interesujaco i technicznie ( live + dzielenie sie wiedza ). 

Wiecej na temat prezentowanych rzeczy mozna poczytac:
1. Tematyka prezentowana przez Tomka [https://learn.microsoft.com/en-gb/azure/deployment-environments/overview-what-is-azure-deployment-environments](https://learn.microsoft.com/en-gb/azure/deployment-environments/overview-what-is-azure-deployment-environments)

2. Tematyk prezentowana przez Emila
https://azurearcjumpstart.io/overview/

Dzięki za spotkanie i do następnego

**Reply by wi5nia** [reply]
Dzieki wszystkim za ciekawe pytania i rozmowy o rozwiazaniu w przerwie :)

**Reply by JanuszNowak** [reply]
@wi5nia usuń śledzenie z linka bo nie działa.
(https://www.linkedin.com/safety/go?url=https%3A%2F%2Flearn.microsoft.com%2Fen-gb%2Fazure%2Fdeployment-environments%2Foverview-what-is-azure-deployment-environments&trk=flagship-messaging-web&messageThreadUrn=urn%3Ali%3AmessagingThread%3A2-YjQzNTUyMjUtMWExZC00MmViLWE1NGQtNjQ3NTQ0MTA0MTAxXzAxMw%3D%3D&lipi=urn%3Ali%3Apage%3Amessaging_thread%3B990ef7b2-ecc1-48ef-84c9-66c3e5c75a08)
```
->
```
(https://learn.microsoft.com/en-gb/azure/deployment-environments/overview-what-is-azure-deployment-environments)
```

**Reply by wi5nia** [reply]
To nie ja wrzucalem linka to co mnie oznaczasz @JanuszNowak :P

**Reply by JanuszNowak** [reply]
@wi5nia a tak widzę :) , że @techitlab 👍, ale już poprawione :)

**Reply by wi5nia** [reply]
Detale robia roznice...

**Reply by kamil-mrzyglod** [reply]
@techitlab Mała poprawka - o Azure Arc mówił Emil, nie Kamil ;)

**Reply by wi5nia** [reply]
A to nie ta sama osoba? 🤔

**Reply by eklime** [reply]
No kto by tam zwracał uwagę na takie rzeczy @kamil-mrzyglod  ;P no i nie miałem slajdu "o mnie", więc w sumie wybaczam :D

**Comment by kkaplukCA** [comment]
Jestem nowy w temacie Azure i bardzo ubolewam nad tym, że Krakowie scena Azure nie istnieje :) Czy można liczyć na nagrania z kolejnych spotkań?

**Reply by mgrabarz** [reply]
Nie planujemy nagrań z kilku powodów, ale rzeczywiście jest to niefajne dla osób spoza Warszawy. 

Skoro scena krakowska nie istnieje, może warto ją zacząć tworzyć. Nikt inny jednak jej nie stworzy niż ktoś na miejscu. Nie trzeba być orłem z Azure by kogoś zaprosić, zakombinować sponsora/lokal. My damy narzędzia, wsparcie  i rozgłos w socjal mediach. To jak? 😈

**Reply by kaluzaaa** [reply]
@kkaplukCA no i oczywiście damy know-how, a jak termin będzie pasował to wpadniemy z wystąpieniami :-)

**Reply by eklime** [reply]
@kkaplukCA podbij do mnie gdzieś na priv. kilka osób też już pytało o Kraków, więc mogę Was połączyć i może coś tam podziałacie. Mogę też dać namiar na lokalnego człowieka, który kiedyś coś tam organizował.

**Reply by kkaplukCA** [reply]
@mgrabarz @kaluzaaa brzmi to jak połowa sukcesu
@eklime a tutaj jak druga połowa. Zostawiam swój namiar na priv i z chęcią włączę się w społeczność i wesprę na bazie waszego know-how i mojego zaangażowania :)

**Comment by aryll** [comment]
Trochę zdjęć z Warszawy 🙃

![1](https://user-images.githubusercontent.com/5950108/196931496-2336bc9a-5689-46bb-9c47-1f8e7d3c69e5.jpg)
![2](https://user-images.githubusercontent.com/5950108/196931502-2c1c0706-4cf9-40ce-a350-01d6b49d2ea4.jpg)
![3](https://user-images.githubusercontent.com/5950108/196931504-0546ce2c-64d0-4398-a87b-0fe86b0c61f7.jpg)
![4](https://user-images.githubusercontent.com/5950108/196931505-bbb64dc7-481c-490c-b51c-6d1183581844.jpg)
![5](https://user-images.githubusercontent.com/5950108/196931506-a4a6e5db-f8e9-4729-9a15-809106cf5069.jpg)
![6](https://user-images.githubusercontent.com/5950108/196931511-1e0a7fec-2447-4c60-99bd-1f355851e131.jpg)
![7](https://user-images.githubusercontent.com/5950108/196931513-cb98815f-37c8-4e91-96c8-0381aed72ea8.jpg)
![8](https://user-images.githubusercontent.com/5950108/196931516-cf521061-14fb-488f-8404-18c0e2643ea2.jpg)
![9](https://user-images.githubusercontent.com/5950108/196931519-b80f912e-d375-4e62-9749-e73550348aa7.jpg)
![10](https://user-images.githubusercontent.com/5950108/196931521-2a5f6eaa-0c9d-4b97-9197-9a88274faa67.jpg)
![11](https://user-images.githubusercontent.com/5950108/196931528-f04cf8d4-c891-4ad9-8fb6-e31c2376ac62.jpg)
